gapic/src/main/com/google/gapid/perfetto/canvas/RenderContext.java

```diff
@@ -272,6 +272,36 @@ private void drawTextLeftTruncate(
     }
   }
 
+  // draws text centered vertically, right truncated to fit into the given width.
```

Comment:
Move new code below drawTextLeftTruncate so that all the left truncate code are together.